css/style.css

```diff
@@ -6,11 +6,10 @@
 /* ===============
    GLOBAL SETTINGS
    =============== */
-* {
-  box-sizing: border-box;
-}
+
 
 body {
+  margin-top: 30px;
```

Comment:
This pushes the slider back down, so it no longer fit the page

Reply:
When I remove it I don't see it change back. is there another issue as well?